src/containers/WatchList/DeleteMovie/container.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Modal } from 'antd'
+import { connect } from 'react-redux'
+import { openNotification } from 'Utils'
+
+import { removeMovieErrorsSelector } from 'Store/watchList/selectors'
+import {
+  removeMovie as removeMovieAction,
+  cleanupRemoveMovieErrors as cleanupRemoveMovieErrorsAction
+} from 'Store/watchList/actions'
+import DeleteMovieComponent from './component'
+
+class DeleteMovie extends Component {
+  onDeleteMovie = () => {
+    const { removeMovie, movieId, title } = this.props
+    return Modal.confirm({
+      title: 'Do you want to delete movie from watchlist?',
+      onOk() { removeMovie(movieId, title) }
```

Comment:
`onOk = () => removeMovie(movieId, title)`

Reply:
1. `onOk: () => removeMovie(movieId, title)` *
2. It doesn’t works this way